microsoft/testsuites/acc/cvm.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from assertpy import assert_that
+
+from lisa import (
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    simple_requirement,
+)
+from lisa.environment import EnvironmentStatus
+from lisa.tools import Dmesg, Lsvmbus
+
+
+@TestSuiteMetadata(
+    area="ACC_CVM",
+    category="functional",
+    description="""
+    This test suite ensures configuration and devices for CVM
+    """,
+)
+class CVMSuite(TestSuite):
+    @TestCaseMetadata(
+        description="""
+        This case verifies that lsvmbus only shows devices
+        that are allowed in a CVM guest
+
+        Steps:
+        1. Call lsvmbus
+        2. Iterate through list returned by lsvmbus to ensure all devices
+           listed are included in valid_class_ids
+        """,
+        priority=1,
+        requirement=simple_requirement(
+            environment_status=EnvironmentStatus.Deployed,
```

Comment:
This is not necessary, it's default value.

Reply:
Alright will remove thank you

Reply:
removed